### PR TITLE
Stage 20.2.1

### PR DESCRIPTION
**Pull request by aradi**
Proposal for hot-fix release. To be merged to `release` and tagged as `20.2.1`

### CODE REVIEW

**Review by bhourahine — COMMENTED** [review]
Probably should include [694f0a2](https://github.com/dftbplus/dftbplus/pull/689/commits/694f0a2ea8c9b917f011566bb32227971c6814ad) as well

**Review by bhourahine — COMMENTED** [review]
Should the manual mention about the last level of the naming convention, perhaps in the releases appendix as the table there won't be updated for 20.2.1

**Comment by aradi** [comment]
I've added the 20.2 release (with the original release date) to the manual.

File: doc/dftb+/manual/dftbp.tex [hunk 1]
```diff
@@ -3119,7 +3119,13 @@ \subsubsection{DftD4 CoordinationNumber}
 \subsubsection{TS}
 \label{sec:dftbp.DispTS}
 
-Applies the Tkatchenko-Scheffler dispersion model~\cite{TkatchenkoPRL09,StohrJCP16}.
+Applies the Tkatchenko-Scheffler dispersion model\
```
Comment (bhourahine) [inline doc/dftb+/manual/dftbp.tex:3122]: There seems to be an extra backslash

**Review by bhourahine — APPROVED** [review]
Looks ready to merge

**Pull request merged**